src/pyOpenMS/pxds/MSExperiment.pxd

```diff
@@ -85,19 +85,19 @@ cdef extern from "<OpenMS/KERNEL/MSExperiment.h>" namespace "OpenMS":
         void resize(Size s) nogil except +
         bool empty() nogil except +
         void reserve(Size s) nogil except +
-        Size getNrSpectra() nogil except +
-        Size getNrChromatograms() nogil except +
+        Size getNrSpectra() nogil except + # wrap-doc:return the number of MS spectra
+        Size getNrChromatograms() nogil except + # wrap-doc:returns the number of chromatograms
         libcpp_vector[unsigned int] getMSLevels() nogil except +  # wrap-ignore
 
-        void sortSpectra(bool sort_mz) nogil except +
+        void sortSpectra(bool sort_mz) nogil except + # wrap-doc:Sort spectra by RT. If sort_mz=True also sort each peak in a spectrum by m/z
         void sortSpectra() nogil except +
```

Comment:
add comments to annotate the difference between sortSpectra(bool sort_mz) and sortSpectra() ?

Reply:
I think it is just an overload because cython doesn't support optional parameters